lib/components/util/styledIcon.tsx

```diff
@@ -72,18 +73,26 @@ export const StyledIconWrapperTextAlign = styled(StyledIconWrapper)<Props>`
   }
 `
 
+const IconButton = styled.div`
+  display: flex;
+  gap: 5px;
+`
```

Comment:
Can we replace the `flex` prop with one that lets you specify a component to use instead of `React.Fragment`? Then we can move this outside of this component and keep it a bit more abstract